src/test/java/domain/LottoNumberTest.java

```diff
@@ -0,0 +1,33 @@
+package domain;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+class LottoNumberTest {
+
+    @Test
+    @DisplayName("로또 숫자는 값이 같다면 동일한 객체를 반환한다")
+    void 로또_숫자는_값이_같다면_동일한_객체를_반환한다() {
+        // given
+        LottoNumber number1 = LottoNumber.from(LottoRule.MIN_LOTTO_NUMBER.getValue());
+        LottoNumber number2 = LottoNumber.from(LottoRule.MIN_LOTTO_NUMBER.getValue());
+
+        // when
+        // then
+        assertThat(number1)
+                .isEqualTo(number2);
+    }
+
+    @Test
+    @DisplayName("로또 숫자는 정해진 범위 사이여야 한다")
+    void 로또_숫자는_정해진_범위가_아니면_예외가_발생한다() {
+        // given
+        // when
+        // then
+        assertThrows(IllegalArgumentException.class, () -> LottoNumber.from(LottoRule.MIN_LOTTO_NUMBER.getValue() - 1));
+        assertThrows(IllegalArgumentException.class, () -> LottoNumber.from(LottoRule.MAX_LOTTO_NUMBER.getValue() + 1));
```

Comment:
```suggestion
        assertAll(
                () -> assertThrows(IllegalArgumentException.class, () -> LottoNumber.from(LottoRule.MIN_LOTTO_NUMBER.getValue() - 1)),
                () -> assertThrows(IllegalArgumentException.class, () -> LottoNumber.from(LottoRule.MAX_LOTTO_NUMBER.getValue() + 1))
        );
```

현재 코드와 assertAll을 사용했을 때의 코드의 차이점은 무엇일까요?

Reply:
확인해 보니 저번 리뷰에서 남겨주신 것과 비슷한 문제가 있는 것 같습니다. 변경 전은 첫 테스트 실패 시 이후의 테스트를 무시하고, assertAll로 감싸니 두 테스트 모두 실행 후 `Multiple Failures (2 failures)`이라는 키워드로 모든 테스트가 실패했음을 알려주는 것 같습니다!

감사합니다. 많이 배웠습니다!